web/src/app/services/auth.service.ts

```diff
@@ -53,63 +53,50 @@ export class AuthService {
       );
   }
 
-  formatUserResponse(user, provider): User {
+  formatUserResponse(response, provider): User {
```

Comment:
Now there is `return null`, I think the return type would be `User | null`

Reply:
Thank you. Will change it